docs/esp8266/general.rst

```diff
@@ -151,25 +151,26 @@ SSL/TLS limitations
 ~~~~~~~~~~~~~~~~~~~
 
 ESP8266 uses `axTLS <http://axtls.sourceforge.net/>`_ library, which is one
-of the smallest TLS libraries with the compatible licensing. However, it
+of the smallest TLS libraries with compatible licensing. However, it
 also has some known issues/limitations:
 
 1. No support for Diffie-Hellman (DH) key exchange and Elliptic-curve
-   cryptography (ECC). This means it can't work with sites which force
-   the use of these features (it works ok with classic RSA certifactes).
+   cryptography (ECC). This means it can't work with sites which require
+   the use of these features (it works ok with the typical sites that use
```

Comment:
I'm no longer sure that typical sites nowadays use RSA, or that the statement will be true for much longer, that's why I wrote it like I did.